src/portable/synopsys/dwc2/dcd_dwc2.c

```diff
@@ -646,7 +646,7 @@ static void handle_bus_reset(uint8_t rhport) {
   tu_memclr(xfer_status, sizeof(xfer_status));
 
   _dcd_data.sof_en = false;
-  _dcd_data.allocated_epin_count = 1;
+  _dcd_data.allocated_epin_count = 0;
```

Comment:
is this correct, I thought EP0 also occupies 1 IN ?

Reply:
@hathach Yes, it does. 
 But when we call one by one: 

 1. `dfifo_alloc(rhport, 0x80, CFG_TUD_ENDPOINT0_SIZE);`
 2.  `_dcd_data.allocated_epin_count++;`
 
We have `allocated_epin_count == 2`
after bus_reset and allocation fifo for EP0 IN (if on reset we set it to `allocated_epin_count = 1`)